homeassistant/components/overkiz/cover_entities/vertical_cover.py

```diff
@@ -82,6 +82,7 @@ def current_cover_position(self) -> int | None:
             OverkizState.CORE_CLOSURE,
             OverkizState.CORE_CLOSURE_OR_ROCKER_POSITION,
             OverkizState.CORE_PEDESTRIAN_POSITION,
+            OverkizState.CORE_DEPLOYMENT,
```

Comment:
It would be a 1st one, Somfy recommends us to only use this for horizontal device. IMO the fix should be [here](https://github.com/home-assistant/core/blob/733a2209c2c9efb97a915ae6ce6b1d0940fab521/homeassistant/components/overkiz/cover.py#L25) and add `PergolaHorizontalAwning` as condition.

Reply:
Good one. Let's verify what would be the best way forward here... Especially since the device class is Pergola, instead of Awning.

Reply:
@tetienne would you agree that this fix is valid and can be merged?

Reply:
I still don't like the fact to add deployment here. It will add an edge case Why not apply what I suggest? The device is well an awning.